salt/annotations/config/srv-annotations-config.php

```diff
@@ -32,7 +32,7 @@
         'credential_file' => true,
         'region' => '{{ pillar.elife.aws.region }}',
         {% if pillar.elife.env in ['dev', 'ci'] -%}
-        'endpoint' => 'http://localhost:4100',
+        'endpoint' => 'http://goaws:4100',
```

Comment:
`goaws` is always a container now. This is only used in `dev`\`ci`, nothing risky.